UefiPayloadPkg/UefiPayloadPkg.dsc

```diff
@@ -55,7 +55,7 @@
   # ELF: Build UniversalPayload file as UniversalPayload.elf
   # FIT: Build UniversalPayload file as UniversalPayload.fit
   #
-  DEFINE UNIVERSAL_PAYLOAD            = FALSE
+  DEFINE UNIVERSAL_PAYLOAD            = TRUE

```

Comment:
Do you really want to change the default payload type? maybe use a new commit if you really want to do it.

Reply:
OK , will roll back the default value and use the command to enable the feature.

Reply:
resolved